meshtastic/interfaces/ble/interface.py

```
@@ -1100,24 +724,10 @@ async def log_radio_handler(self, sender: Any, b: bytes | bytearray) -> None:
         self._log_radio_handler(sender, b)
 
     def _legacy_log_radio_handler(self, _: Any, b: bytes | bytearray) -> None:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_`

(ANN401)